lms/djangoapps/learner_recommendations/serializers.py

```diff
@@ -0,0 +1,43 @@
+"""
+Serializers for course about page recommendations.
+"""
+from rest_framework import serializers
+
+
+class ActiveCourseRunSerializer(serializers.Serializer):
+    """Serializer for active course run for course about page recommendations API"""
+    key = serializers.CharField()
+    type = serializers.CharField()
+    marketingUrl = serializers.URLField(source="marketing_url")
+
+
+class CourseOwnersSerializer(serializers.Serializer):
+    """Serializer for course owners for course about page recommendations API"""
+    key = serializers.CharField()
+    name = serializers.CharField()
+    logoImageUrl = serializers.URLField(source="logo_image_url")
+
+
+class RecommendedCourseSerializer(serializers.Serializer):
+    """Serializer for a recommended course from the recommendation engine"""
+
+    uuid = serializers.CharField()
+    title = serializers.CharField()
+    image = serializers.CharField()
+    prospectusPath = serializers.CharField(source="prospectus_path")
+    owners = serializers.ListField(
+        child=CourseOwnersSerializer(), allow_empty=True
+    )
+    activeCourseRun = ActiveCourseRunSerializer(source="active_course_run")
+
+
+class CourseRecommendationSerializer(serializers.Serializer):
```

Comment:
this is missing program upsell

Reply:
That will be handled once Attiya's PR is merged.